src/frontends/pytorch/src/op/complex.cpp

```diff
@@ -17,12 +17,23 @@ namespace op {
 
 using namespace ov::op;
 
+OutputVector translate_real(const NodeContext& context) {
+    num_inputs_check(context, 1, 1, true);
+    auto complex = context.get_input(0);
+
+    auto complex_type_mark = as_type_ptr<ComplexTypeMark>(complex.get_node_shared_ptr());
+    if (!complex_type_mark)
+        // if input is not a complex number, just return it as is
+        return {complex};
+    return {complex_type_mark->get_real()};
+};
+
```

Comment:
please remove it and introduce necessary changes in common translator. This addition will not impact other FEs